json_rpc/src/filters/tests/main_filter_with_recovery_tests.rs

```diff
@@ -219,7 +221,7 @@ async fn should_handle_request_with_extra_field() {
     // This should get handled by `filters::handle_body` and return Response::Failure (invalid
     // request) to the client as the ID has fractional parts.
     let http_response = warp::test::request()
-        .body(r#"{"jsonrpc":"2.0","id":1,"method":"get good thing","params":2,"extra":"field"}"#)
+        .body(r#"{"jsonrpc":"2.0","id":1,"method":"get good thing","params":[2],"extra":"field"}"#)
```

Comment:
The comment says that id has fractional parts but it's not true. This one is incorrect as it has an extra field?

Reply:
Yup - good catch - thanks!

Reply:
Fixed in [a11b164](https://github.com/casper-network/casper-node/pull/3155/commits/a11b164bdc53b713db4f181e77bdcef32c33fae2).